src/core/room-status.ts

```diff
@@ -194,11 +193,12 @@ export class DefaultRoomLifecycle extends EventEmitter<RoomStatusEventsMap> impl
    * @inheritdoc
    */
   onChange(listener: RoomStatusListener): StatusSubscription {
-    this.on(listener);
+    listener = wrap(listener);
```

Comment:
Nitpick: I think its cleaner to do something like:

`const wrapped = wrap(listener)` rather than reassigning

Reply:
(And in other places)

Reply:
done